delivery.yaml

```diff
@@ -30,5 +30,8 @@ pipeline:
         cmd: |
           ./gradlew clean app:bootJar
           IMAGE="registry-write.opensource.zalan.do/aruha/nakadi-oss:${CDP_BUILD_VERSION}"
```

Comment:
Maybe you always want to build "pr" version when it's merged to master because in the deployment of staging I believe you'll always deploy from pr image and you may want to deploy a "pr" image that is just like the "master" image but with a different name.

Reply:
you might want to name it "test" for this reason, because it's not always "pr" since it can be built from master but still be named "test" just for the sake of simplicity in the deployment part of the pipeline.